droidlet/memory/memory_nodes.py

```diff
@@ -861,45 +903,53 @@ def update_condition(self, conditions):
             setattr(self.task, k, condition)
         self.update_task()
 
-    # FIXME TODO don't need paused, set prio to 0 and have a condtion for unpausing
+    # FIXME names/constants for some specific prios
     # use this to update prio or running, don't do it directly on task or in db!!
     def get_update_status(self, status, force_db_update=True, force_task_update=True):
         """
-        status is a dict with possible keys "prio", "running", "finished".
+        status is a dict with possible keys "prio", "running", "paused", "finished".
 
         prio > 0  :  run me if possible, check my stop condition
         prio = 0  :  check my run_condition, run if true
         prio = -1 :  check my init_condition, set prio = 0 if True
-        prio < -1 :  don't even check init_condition or run_condition, I'm done
+        prio < -1 :  don't even check init_condition or run_condition, I'm done or unhatched
```

Comment:
nit: add comment for prio = -2

Reply:
in another PR I will make constants for all of these, with documentation.  see FIXME on line 906